test/controllers/disk_controller_test.rb

```diff
@@ -1,24 +1,31 @@
 require "test_helper"
 require "database/setup"
 
-require "active_storage/disk_controller"
-
-class ActiveStorage::DiskControllerTest < ActionController::TestCase
+class ActiveStorage::DiskControllerTest < ActionDispatch::IntegrationTest
   setup do
     @blob = create_blob
-    @routes = Routes
-    @controller = ActiveStorage::DiskController.new
   end
 
   test "showing blob inline" do
-    get :show, params: { filename: @blob.filename, encoded_key: ActiveStorage.verifier.generate(@blob.key, expires_in: 5.minutes, purpose: :blob_key) }
-    assert_equal "inline; filename=\"#{@blob.filename}\"", @response.headers["Content-Disposition"]
+    get rails_disk_blob_url(
+      filename: "hello.txt",
+      content_type: @blob.content_type,
```

Comment:
Here I'd to pas content-length in order for tests to pass. @dhh  I've noticed that you changed the approach [Here](https://github.com/rails/activestorage/commit/5944850bc1259ca42381ce83d155ddd914b968c6#diff-a8a2fcba8c0f485c1902e82082c3d582R15).

Not sure if your intention was to pass content_type of to figure him out automatically ?